modules/ossm-migrating-read-me-explicitly-create-openshift-routes.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-assembly.adoc
+
+//Start of an overall Migrating section.
+//Section is most likely to be reworked/restructured with OSSM 2 to OSSM 3 migration guides for GA. Unknown how many migration guides there are at this time (11/11/2024). It would be beneficial to be able to link from differences to the relevent migration guide so that users A) understand the change, esp significant changes like new operator, installing tracing and Kiali separately, gateways, etc.
+
+:_mod-docs-content-type: CONCEPT
+[id="explicitly-create-openshift-routes_{context}"]
+= Explicitly create OpenShift Routes
+//In the ocp-docs repo, there is no attribute for OpenShift Container Platform Ingress Operator so followed https://docs.openshift.com/container-platform/4.17/networking/ingress-operator.html
+An OpenShift `Route` resource allows an application to be exposed with a public URL using {product-title} Ingress Operator for managing HAProxy based Ingress controllers.
+
+{SMProduct} 2 used Istio OpenShift Routing (IOR) that automatically created and managed OpenShift routes for Istio gateways. While this was convenient, as the operator managed these routes for the user, it also caused confusion around ownership as many `Route` resources are managed by administrators. OpenShift Routing also lacked the ability to configure an independent `Route` resource, created unnecessary routes, and exhibited unpredictable behavior during upgrades.
```

Comment:
```suggestion
{SMProduct} 2 used Istio OpenShift Routing (IOR) that automatically created and managed OpenShift routes for Istio gateways. While this was convenient, as the operator managed these routes for the user, it also caused confusion around ownership as many `Route` resources are managed by administrators. Istio OpenShift Routing also lacked the ability to configure an independent `Route` resource, created unnecessary routes, and exhibited unpredictable behavior during upgrades.
``` The "Istio" part there is important, because "OpenShift Routing" is the still supported OCP feature.